src/sfctl/helps/resource.py

```diff
@@ -0,0 +1,93 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -----------------------------------------------------------------------------
+
+"""Help documentation for managing Service Fabric Mesh Resources."""
+
+from knack.help_files import helps
+
+helps['mesh app show'] = """

```

Comment:
Since these are not custom commands, you don't need to add the helps section (unless for some reason the help text doesn't show up without this file - in such a case, we should investigate why that is)

Reply:
I have updated the PR checklist page to make this more clear. Sorry for the confusion.

Reply:
Actually, this file would be quite good to have - we don't need the help text here, but we should be adding examples for all the commands, especially if any of them has any parameters which may be confusing.